exporters/metric/cortex/cortex.go

```diff
@@ -91,7 +92,7 @@ func NewExportPipeline(config Config, options ...push.Option) (*push.Controller,
 	}
 
 	pusher := push.New(
-		simple.NewWithExactDistribution(),
+		simple.NewWithHistogramDistribution(config.HistogramBoundaries),
```

Comment:
I don't think you need to address it in this PR, but I'm not a huge fan of the package name `simple`. Having a `simple.go` file that defines the simple selector in the `selector` package would be more clear I think. `selector.NewSimpleWithHistogramDistribution...`.

Reply:
I agree it's not super clear. It's a package from the SDK, so I'll express that concern to the Go SIG.